README.md

```diff
@@ -29,7 +29,6 @@ Docker image publicly available on Docker Hub: https://hub.docker.com/r/cfplatfo
 |`repository.authenticate`   |out     |optional         | set to `in` to authenticate to a private repo for check and download only, `always` to authenticate to a private repository for upload, check and download.
 
 ### Deprecated parameters (since version 0.2.0)
-* `python_version`: gets mapped to `filename_match` if it's not a version number. `python_version` is now only used for the actual interpreter version to have a transparent mapping to pip.
```

Comment:
Not really deprecated according to the [code](https://github.com/cf-platform-eng/concourse-pypi-resource/blob/67c7d902ed0b0429f3bc68a86ef8870b2216aa39/pypi_resource/common.py#L76). 
```
    deprecated_keys = {
        'username',
        'password',
        'repository_url',
        'test',
    }
```

Reply:
Actually, I was wrong. This is still correct. I will revert this commit.